_templates/cloudflare/pages-tinygo/pages/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+  </head>
+  <body>
+    <p>
+      <a href="/api/hello">/api/hello</a>
+    </p>
+    <p>
+      <a href="/api/hello?name=syumai">/api/hello?name=Example</a>
```

Comment:
I want to remove the value `syumai` from this template
```suggestion
      <a href="/api/hello?name=Example">/api/hello?name=Example</a>
```

Reply:
Sorry about this! I thought I had changed it! should be good now.